CMakeLists.txt

```diff
@@ -149,6 +149,10 @@ ELSE()
   ADD_COMPILE_FLAG("-Wall")
   ADD_COMPILE_FLAG("-Werror")
   ADD_COMPILE_FLAG("-Wextra")
+  execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpversion OUTPUT_VARIABLE GCC_VERSION)
```

Comment:
Wil this work if `CMAKE_C_COMPILER` is cl.exe (i.e. windows)?

Reply:
It should work as this was on the "not Windows" condition branch.